src/Events/Controllers/AppController.cs

```diff
@@ -83,13 +84,14 @@ public async Task<ActionResult<string>> Post([FromBody] CloudEventRequestModel c
 
             try
             {
-                string cloudEventId = await _eventsService.SaveAndPostInbound(_mapper.Map<CloudEvent>(cloudEvent));
+                string cloudEventId = await _eventsService.PostRegistration(_mapper.Map<CloudEvent>(cloudEvent));
+                _logger.LogInformation("Cloud Event successfully registered with id: {cloudEventId}", cloudEventId);
```

Comment:
Do we need to log the happy case here? Or was it just used for testing?

Reply:
Removed.